code/modules/mob/emote.dm

```diff
@@ -52,7 +52,9 @@
 						M.show_message(message, m_type)
 						viewers.Add(M)
 				O.show_message(message, m_type)
-			langchat_speech(input, viewers, GLOB.all_languages, skip_language_check = TRUE, additional_styles = list("langchat_italic"))
+				if(!(O?.client?.prefs.toggles_langchat & LANGCHAT_SEE_EMOTES))
```

Comment:
O is definitely always valid here, so remove the ?. after O.
Also this will probably runtime if you try to do a bit operation with a null value, so consider storing it as a variable first and checking the validity of the variable before doing the bit operation